index.html

```diff
@@ -1,184 +1,356 @@
+                    ><img src="/img/medium icon.svg" alt="medium icon"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="/img/github icon.svg" alt="github icon"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="/img/hi icon.svg" alt="Hi icon" /></a>
+                </li>
+              </ul>
             </div>
-        </section>    
+            <button class="btn btn-primary">Get My Resume</button>
+          </div>
+          <div class="skils">
+            <ul class="skills-items">
+              <li class="skill-item">
+                <h3 class="lang">
+                  Languages <img src="/img/down arrow.png" alt="" />
+                </h3>
+                <br />
+                <span class="item">
+                  <img src="/img/js.png" alt="" />
+                  <h4>Javascript</h4></span
+                >
+                <br />
+                <span class="item">
+                  <img src="/img/html.png" alt="" />
+                  <h4>HTML</h4></span
+                >
+                <br />
+                <span class="item">
+                  <img src="/img/css.png" alt="" />
+                  <h4>CSS</h4></span
+                >
+                <br />
+              </li>
+              <li class="skill-item">
+                <h3 class="lang">
+                  Frameworks <img src="/img/Union.png" alt="" />
+                </h3>
+              </li>
+              <hr />
+              <li class="skill-item">
+                <h3 class="lang">Skills <img src="/img/Union.png" alt="" /></h3>
+              </li>
+              <hr />
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section id="contact-section">
+        <div class="contact-container">
+          <div class="contact-decs">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <div class="contact-form">
+            <form
+              id="fs-frm"
+              name="simple-contact-form"
+              accept-charset="utf-8"
+              action="https://formspree.io/f/mdojgyyj"
+              method="post"
+            >
+              <fieldset id="fs-frm-inputs">
+                <input
+                  type="text"
+                  name="name"
+                  id="full-name"
+                  required=""
+                  placeholder="Seko"
+                  maxlength="30"
+                />
+
+                <input
+                  type="email"
+                  name="_replyto"
+                  id="email-address"
+                  required=""
+                  placeholder="sekodameh@gmail.com"
```

Comment:
[OPTIONAL] I highly recommend you change your placeholders to something more generic, like "Name" and "example@email.com". This will improve your user experience and avoid misunderstandings 😉.